src/core/framelesswindowsmanager.cpp

```diff
@@ -97,7 +98,8 @@ bool FramelessWindowsManagerPrivate::usePureQtImplementation()
         if (qEnvironmentVariableIntValue(kUsePureQtImplFlag) != 0) {
             return true;
         }
-        const QString iniFilePath = QCoreApplication::applicationDirPath() + QChar(u'/') + kConfigFileName;
```

Comment:
A fixing to avoid following errors:
```
[8/55 2.5/sec] Building CXX object src\core\CMakeFiles\FramelessHelperCore.dir\framelesswindowsmanager.cpp.obj
FAILED: src/core/CMakeFiles/FramelessHelperCore.dir/framelesswindowsmanager.cpp.obj 
"D:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\bin\HostX64\x64\cl.exe"  /nologo /TP -DFRAMELESSHELPER_CORE_LIBRARY -DFramelessHelperCore_EXPORTS -DNOMINMAX -DNTDDI_VERSION=0x0A00000B -DQT_CORE_LIB -DQT_DEPRECATED_WARNINGS -DQT_DISABLE_DEPRECATED_BEFORE=0x060400 -DQT_GUI_LIB -DQT_NO_CAST_FROM_ASCII -DQT_NO_CAST_FROM_BYTEARRAY -DQT_NO_CAST_TO_ASCII -DQT_NO_FOREACH -DQT_NO_KEYWORDS -DQT_NO_NARROWING_CONVERSIONS_IN_CONNECT -DQT_NO_URL_CAST_FROM_STRING -DQT_USE_QSTRINGBUILDER -DUNICODE -DWIN32_LEAN_AND_MEAN -DWINRT_LEAN_AND_MEAN -DWINVER=0x0A00 -D_CRT_NON_CONFORMING_SWPRINTFS -D_CRT_SECURE_NO_WARNINGS -D_ENABLE_EXTENDED_ALIGNED_STORAGE -D_UNICODE -D_WIN32_IE=0x0A00 -D_WIN32_WINNT=0x0A00 -IX:\dev\build-framelesshelper-2.0-Desktop_x86_windows_msvc2019_pe_64bit-Debug\src\core\FramelessHelperCore_autogen\include -IX:\dev\framelesshelper-2.0\src\core\..\..\include\FramelessHelper\Core -IX:\Qt\vs2019_x64_5.15.2_install\include\QtCore\5.15.2 -IX:\Qt\vs2019_x64_5.15.2_install\include\QtCore\5.15.2\QtCore -IX:\Qt\vs2019_x64_5.15.2_install\include -IX:\Qt\vs2019_x64_5.15.2_install\include\QtCore -IX:\Qt\vs2019_x64_5.15.2_install\.\mkspecs\win32-msvc -IX:\Qt\vs2019_x64_5.15.2_install\include\QtGui\5.15.2 -IX:\Qt\vs2019_x64_5.15.2_install\include\QtGui\5.15.2\QtGui -IX:\Qt\vs2019_x64_5.15.2_install\include\QtGui -IX:\Qt\vs2019_x64_5.15.2_install\include\QtANGLE /DWIN32 /D_WINDOWS /EHsc /Zi /Ob0 /Od /RTC1 -MDd /utf-8 /W4 /WX -std:c++17 /showIncludes /Fosrc\core\CMakeFiles\FramelessHelperCore.dir\framelesswindowsmanager.cpp.obj /Fdsrc\core\CMakeFiles\FramelessHelperCore.dir\ /FS -c X:\dev\framelesshelper-2.0\src\core\framelesswindowsmanager.cpp
X:\Qt\vs2019_x64_5.15.2_install\include\QtCore/qstringbuilder.h(120): error C2220: 以下警告被视为错误
X:\Qt\vs2019_x64_5.15.2_install\include\QtCore/qstringbuilder.h(120): warning C4127: 条件表达式是常量
X:\Qt\vs2019_x64_5.15.2_install\include\QtCore/qstringbuilder.h(120): note: 考虑改用 "if constexpr" 语句
X:\Qt\vs2019_x64_5.15.2_install\include\QtCore/qstringbuilder.h(131): note: 查看对正在编译的函数 模板 实例化“T QStringBuilder<QStringBuilder<QString,QChar>,QString>::convertTo<QString>(void) const”的引用
        with
        [
            T=QString
        ]
X:\Qt\vs2019_x64_5.15.2_install\include\QtCore/qstringbuilder.h(131): note: 查看对正在编译的函数 模板 实例化“T QStringBuilder<QStringBuilder<QString,QChar>,QString>::convertTo<QString>(void) const”的引用
        with
        [
            T=QString
        ]
X:\Qt\vs2019_x64_5.15.2_install\include\QtCore/qstringbuilder.h(131): note: 在编译 类 模板 成员函数“QStringBuilder<QStringBuilder<QString,QChar>,QString>::operator QString(void) const”时
X:\dev\framelesshelper-2.0\src\core\framelesswindowsmanager.cpp(100): note: 查看对正在编译的函数 模板 实例化“QStringBuilder<QStringBuilder<QString,QChar>,QString>::operator QString(void) const”的引用
X:\dev\framelesshelper-2.0\src\core\framelesswindowsmanager.cpp(100): note: 查看对正在编译的 类 模板 实例化“QStringBuilder<QStringBuilder<QString,QChar>,QString>”的引用
```